packages/react/src/utils/mergeThemes.ts

```diff
@@ -144,6 +144,46 @@ export const mergeComponentStyles__DEV = (
   }, initial)
 }
 
+const mergeComponentStylesCache = new WeakMap()
+
+const containsFunctionProp = obj => {
```

Comment:
this doesn't make sense, as if all of the params are not functions, we won't even need merging, although we are merging and converting them to functions now..